kasa/aestransport.py

```diff
@@ -28,6 +27,7 @@
     SmartErrorCode,
     TimeoutException,
 )
+from .httpclientsession import HttpClientSession
```

Comment:
Minor nit: http_client_session

Reply:
Do you mean for the module name?  I've kept this consistent with all the other module names which do not have underscores (except the `device_factory` and `device_type` which do).

Reply:
Yeah it's inconsistent. I think either way is fine